### DISCUSSION TITLE
Auto-Merge for specific PRs Authors

### DISCUSSION THREAD

**Discussion by eyadashrafkh** [community/community#130510] [Actions] [answered]
### Select Topic Area

Question

### Body

If a repo has many PRs that the owner can't review all the PRs, so the owner assigned few people to review (with read-only access)the PRs and LGTM the correct PRs so he needs to make an action that allows auto-merge for those reviewer when they comment "LGTM" or whatever the comment. I tried to search for a code that performs that as i am not a CI/CD expert and of course asked ChatGPT for code but it wasn't correct. So is there a documentation that i can read to understand the config of .yml to perform such a feature.
Also can i trigger this action for PRs from a trusted authors too ??
Thanks in advance and sorry if the question was not professional enough, redundant question and bad formatted / documented.

FYI: i got cancelled once on stackoverflow because i asked a question that was really messed up XD.

**Comment by davevad93** [comment, marked as answer]
Hi @eyadashrafkh ,

Take a look at the "***Automatically enable auto-merge for pull requests from trusted authors***" section of this guide:

- [How to automatically merge trusted Pull Requests on GitHub](https://jhale.dev/posts/auto-merging-prs/)

**Reply by grandmas-favorite** [reply]
Nice one @davevad93! 😄
